src-ui/lib/components/BooksTableRow.svelte

```diff
@@ -1,13 +1,21 @@
 <script lang="ts">
-  import type { Book } from '../types/book.js'
+  import type { Book } from '../../lib/types/book.js'
+
   export let book: Book
+  export let handleEdit: (book: Book, field: keyof Book, event: Event) => void
 </script>
 
 <tr>
-  <td>{book.isbn10 ?? book.isbn13}</td>
-  <td>{book.title || ''} </td>
-  <td>{book.authors || ''}</td>
-  <td>{book.tags || ''}</td>
+  <td contenteditable="true" on:blur={(e) => handleEdit(book, 'isbn10', e)}
```

Comment:
Fine to merge as-is, but I wonder if it's possible to make `contenteditable="true" on:blur={(e) => handleEdit(book, 'isbn10', e)` cleaner. Any ideas / do you think it's worth the squeeze?